Graphics/Blank/JavaScript.hs

```diff
@@ -624,7 +618,17 @@ jsEscapeChar '\t' = "\\t"
 jsEscapeChar '\v' = "\\v"
 jsEscapeChar '\"' = "\\\""
 jsEscapeChar '\'' = "\\'"
--- Non-control ASCII characters can remain as they are.
-jsEscapeChar c' | not (isControl c') && isAscii c' = TL.singleton c'
--- All other non ASCII signs are escaped to unicode.
-jsEscapeChar c' = jsUnicodeChar c'
+-- Code borrowed from GHCJS implementation: https://github.com/ghcjs/ghcjs/blob/718b37fc7167269ebca633914c716c3dbe6d0faf/src/Compiler/JMacro/Base.hs#L887-L905
+jsEscapeChar '/'  = "\\/"
+jsEscapeChar c
+    -- Non-control ASCII characters can remain as they are.
+    | not (isControl c) && isAscii c = TL.singleton c
```

Comment:
I noticed that there is a slight difference between this version of the function and the GHCJS one. The former function uses `isAscii`, while the latter uses `ord c <= 127`. Are these always guaranteed to be the same thing? (I am the furthest thing in the world from a Unicode expert, so I apologize if this is a dumb question.)

Reply:
Yes, ASCII is exactly first 128 characters (from 0 up to 127). `isAscii` is just a better name here, I think, since this is what logically happens: we only print non-control ASCII characters as is.

Reply:
BTW, I'm also far from being an expert in Unicode. Probably half of what I know about it or more, I learned just digging into this issue with symbols not displaying properly with codeworld when run locally :)

Reply:
Makes sense. I wonder if it would be worth changing the upstream GHCJS code to use `isAscii`?